bambou/session.go

```diff
@@ -460,7 +501,7 @@ func (s *Session) NextEvent(channel NotificationsChannel, lastEventID string) *E
 
 	request, err := http.NewRequest("GET", currentURL, nil)
 	if err != nil {
-		return NewError(http.StatusBadRequest, err.Error())
+		return NewBambouError("", err.Error())
```

Comment:
Why are we not returning the bad request code as title? Not a big fan of empty titles